src/redux/dragons/dragon.js

```diff
@@ -0,0 +1,49 @@
+const FETCH_DRAGONS = 'FETCH_DRAGONS';
+const ADD_RESERVATION = 'ADD_RESERVATION';
+const CANCEL_RESERVATION = 'CANCEL_RESERVATION';
+const URL = 'https://api.spacexdata.com/v3/dragons';
+
+export const fetchDragons = async (dispatch) => {
+  const response = await fetch(URL);
+  const data = await response.json();
+  dispatch({ type: FETCH_DRAGONS, payload: data });
+};
```

Comment:
- [ ] After fetching the dragons, you are still required to only store the needed data in the store. Right now you are storing all data. Kindly fix that. You should only save these attributes: `id, name, type, flickr_images`

  Dragons state | Mission state
  :---:|:---:
  ![drgon-1](https://user-images.githubusercontent.com/81468675/187947599-58d4db2e-982c-4300-9d76-db096c140772.jpg) | ![mission](https://user-images.githubusercontent.com/81468675/187950122-0d712566-417c-489a-9432-179081b9e238.jpg)